libcodechecker/libhandlers/analyze.py

```diff
@@ -22,6 +22,7 @@
 from libcodechecker.logger import LoggerFactory
 
 LOG = LoggerFactory.get_new_logger('ANALYZE')
+CTU_FUNC_MAP_CMD = 'clang-func-mapping'
```

Comment:
This default variable should be moved into a config variable, created by `package_context`, and read from `config\package_layout.json`.